src/TumblThree/TumblThree.Applications/Downloader/AbstractDownloader.cs

```diff
@@ -289,7 +289,7 @@ protected virtual async Task<bool> DownloadBinaryPostAsync(TumblrPost downloadIt
                 string fileName = FileName(downloadItem);
                 UpdateProgressQueueInformation(Resources.ProgressSkipFile, fileName);
             }
-            else if (!shellService.Settings.LoadAllDatabases && blog.CheckDirectoryForFiles && (blog.CheckIfBlogShouldCheckDirectory(FileNameUrl(downloadItem), FileNameNew(downloadItem))
+            /*else if (!shellService.Settings.LoadAllDatabases && blog.CheckDirectoryForFiles && (blog.CheckIfBlogShouldCheckDirectory(FileNameUrl(downloadItem), FileNameNew(downloadItem))
```

Comment:
Why are you removing functionality here?

Reply:
Invisible options are a hidden danger. I removed it to reduce testing noise since it `AddFileToDb`, this should probably be reverted but needs to be retested/rewritten.